src/Components/QuizDetails/QuizQuestions/QuestionsHeader.tsx

```diff
@@ -0,0 +1,63 @@
+import { Grid, Typography, Button } from "@mui/material";
+import React from "react";
+import { GetApp, AddCircle } from "@mui/icons-material";
+
+export const QuestionsHeader = ({ children }: React.PropsWithChildren) => {
+  return (
+    <Grid
```

Comment:
We can think about extracting components for each item here to have better separation and readability.